CONTRIBUTING.md

```diff
@@ -53,6 +53,33 @@ If you have any questions or need assistance, don't hesitate to reach out to us
 
 Happy Hacking!
 
+
+# Code Review Policy
+
+* CI must pass (enforced)
+* 1 review is mandatory (enforced), 2 or more ideal
+* If you believe your change is simple, and non-controversial enough, and you want
+  to avoid merge conflicts, or blocking work before it gets enough reviews, label it with
+  `needs further review` label and Merge it.
```

Comment:
label was called something "futher review" right

Reply:
@maan2003 I don't understand. After a very brief chat we decided to go with `nneds further review` with Eric, since we are using lower cases . Do you want `further review`? IMO, not self explanatory enough, but I don't feel very strongly about it.

Reply:
oh I just didn't read https://github.com/fedimint/fedimint/pull/4948#discussion_r1563135622